lib/project/pro_motion/table.rb

```diff
@@ -1,7 +1,8 @@
 module ProMotion
   class TableScreen < TableViewController
-    # For some reason, this empty class definition is required
-    # or we get a NoMethodError with the `searchable`
-    # class method call. O_o
+    # Don't call super -- would result in two on_load calls to the cell.
+    def on_cell_created(cell, data)
+      build(cell)
+    end
```

Comment:
is this something that we will be overriding in our tables often?  I ask because we have data here but are not doing anything with it - wondering if there is something we can do here to leverage that extra parameter.  Based on the comments in the PM PR I'm guessing thats the case - but would we really want to not call super?  Or is this something similar to #91 that we need to manage properly?